<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWorkplaceDialog</class>
 <widget class="QDialog" name="EditWorkplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>IP:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Заголовок</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="3">
    <widget class="QLineEdit" name="lineEditVNCPass">
     <property name="text">
      <string>88888888</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>VNC Port:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>VNC Pass:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QSpinBox" name="spinBoxVNCPort">
     <property name="maximum">
      <number>99999</number>
     </property>
     <property name="value">
      <number>5900</number>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <widget class="QLineEdit" name="lineEditIP"/>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QComboBox" name="comboBoxArmType">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Pos ID:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QSpinBox" name="spinBoxPosID">
     <property name="suffix">
      <string/>
     </property>
     <property name="prefix">
      <string> </string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Тип Модуля</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="labelTypeName">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxArmType</tabstop>
  <tabstop>spinBoxPosID</tabstop>
  <tabstop>lineEditIP</tabstop>
  <tabstop>spinBoxVNCPort</tabstop>
  <tabstop>lineEditVNCPass</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
